src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -20,6 +21,24 @@ function setOrderFormData(response) {
   }
 }
 
+function setPartialPaymentOrderObject(response) {
+  const partialPaymentsOrder = {
```

Comment:
Why make an exception in structure for these two variables?

Reply:
I left it like that because on one place this object is being used as requestData. @zaiddreakh do you have more input on this?

Reply:
I restructured the object as discussed.